lib/adapter.js

```diff
@@ -56,6 +56,15 @@ class JasmineAdapter {
             return true
         }
 
+        const stopOnSpecFailure = this.getStopOnSpecFailure()
+        if (stopOnSpecFailure) {
+            try {
+                jasmine.getEnv().stopOnSpecFailure(stopOnSpecFailure)
+            } catch (e) {
+                console.log('Warning: installed version of Jasmine doesn\'t support stopOnSpecFailure')
```

Comment:
We pin the version if the framework adapter usually to a jasmine version, so no reason to check support for this

Reply:
makes sense